resources/html/test/testPlayground.html

```diff
@@ -0,0 +1,188 @@
+<!--
```

Comment:
I have an idea. Let's put a comment above each div block to indicate what System Tests are using this div. This would make it much easier to find what the tests are testing against.